docs/developers/useful-info.md

```diff
@@ -1,11 +1,15 @@
 ---
 description: Network information and deployed contracts
-sidebar_position: 2
+sidebar_position: 3
 ---
 
 # Network information and deployed contracts
 
-## Network Information
+## Network status
+
+You can find out how to navigate to Linea network status [here](../reference/status.md).
```

Comment:
This sentence is very unclear. Do you mean "You can find out more about the current status of the Linea network by navigating [here]"?